cli/pkg/otel/otel-utilities.go

```diff
@@ -46,53 +46,138 @@ func CreateCertManager(confirmActions bool) error {
 				utils.ConfirmAction()
 			}
 
-			res, err := http.Get(CertManagerManifests)
+			err = createUpgradeCertManager()
 			if err != nil {
-				return fmt.Errorf("failed to download %v", err)
+				return fmt.Errorf("failed to create cert-manager %v", err)
 			}
-			defer res.Body.Close()
+			fmt.Println("Successfully created cert-manager")
+			return nil
+		}
+	}
+	fmt.Println("cert-manager exists in the cluster, skipping the installation...")
```

Comment:
When running tobs against a cluster with already installed cert-manager, it would be good to check if correct version of Certificates CRD is available (if API is available). Previously we only checked (L41) if API is available, but since cert-manager deprecated some of API versions, that check is too weak now.